pkgs/tools/security/truffleHog/default.nix

```diff
@@ -0,0 +1,38 @@
+{ lib, buildPythonApplication, buildPythonPackage, fetchPypi, GitPython }:
+
+let
+  truffleHogRegexes = buildPythonPackage rec {
+    pname = "truffleHogRegexes";
+    version = "0.0.4";
+    src = fetchPypi {
+      inherit pname version;
+      sha256 = "09vrscbb4h4w01gmamlzghxx6cvrqdscylrbdcnbjsd05xl7zh4z";
+    };
+  };
+in
+  buildPythonApplication rec {
+    pname = "truffleHog";
+    version = "2.0.91";
+
+    src = fetchPypi {
+      inherit pname version;
+      sha256 = "0r4c9ihy6wjh5cwli7lb6cr2yfvxrh7r6cgznql1src5gzlnkymx";
+    };
+
+    # Relax overly restricted version constraint
+    postPatch = ''
+      sed -i "s/GitPython == 2.1.1/GitPython >= 2.1.1/" setup.py
+    '';
```

Comment:
CC @FRidh is there a better way to loosen overly conservative semver bounds?  I did build all of these packages, and verified that truffleHog is working fine with the newer GitPython.

Reply:
Sadly, there isn't. But using
```
sed -i "s/GitPython ==/GitPython >=/" setup.py
```
could reduce the pain upon updating.
Also, consider using `substituteInPlace`.